tests/ui/crashes/ice-4727-aux.rs

```diff
@@ -0,0 +1,13 @@
+//ignore-test
```

Comment:
You can move this file in the auxiliary dir, so you don't have to ignore it, like 

https://github.com/rust-lang/rust-clippy/blob/43a3796379e59a09d7de7b8e441f4a166737e568/tests/ui/auxiliary/use_self_macro.rs#L1

which is used in

https://github.com/rust-lang/rust-clippy/blob/43a3796379e59a09d7de7b8e441f4a166737e568/tests/ui/ice-4671.rs#L4-L5